packages/pool/contracts/StateUtils.sol

```diff
@@ -188,6 +196,19 @@ contract StateUtils is IStateUtils {
         emit SetDaoAgent(daoAgent);
     }
 
+    /// @notice Called after deployment to set the addresses of the DAO Voting
+    /// apps
+    /// @param _votingApps Addresses of the DAO Api3Voting apps
+    function setVotingApps(address[] calldata _votingApps)
+        external
+        override
+    {
+        require(_votingApps.length != 0, ERROR_VALUE);
+        require(votingApps.length == 0, ERROR_UNAUTHORIZED);
```

Comment:
Should we limit the number of Voting Apps to two, or you are thinking it could be even more in the future?

Reply:
I suppose we can limit it at 2 because you set them once and wouldn't be able to update afterwards. And then, I think it would make more sense to unroll that array as `votingApp1` and `votingApp2`, what do you think?

Reply:
I just created an issue, let's deal with it later https://github.com/api3dao/api3-dao/issues/147

Reply:
> I suppose we can limit it at 2 because you set them once and wouldn't be able to update afterwards. And then, I think it would make more sense to unroll that array as `votingApp1` and `votingApp2`, what do you think?

makes sense, let's merge this PR, so we can deal with this part later on